.github/workflows/crux-mir-build.yml

```diff
@@ -136,13 +136,22 @@ jobs:
         run: .github/ci.sh test crux-mir
 
       - shell: bash
-        run: .github/ci.sh bundle_crux_mir_files
+        run: |
+          NAME="crux-mir-${{ needs.outputs.outputs.crux-mir-version }}-${{ matrix.os }}-x86_64"
+          echo "NAME=$NAME" >> $GITHUB_ENV
+          .github/ci.sh bundle_crux_mir_files
         if: github.repository == 'GaloisInc/crucible'
         env:
           OS_TAG: ${{ matrix.os }}
+          VERSION: ${{ needs.outputs.outputs.crux-mir-version }}
+
+      - if: github.event.pull_request.head.repo.fork == false
+        shell: bash
+        env:
           SIGNING_PASSPHRASE: ${{ secrets.SIGNING_PASSPHRASE }}
           SIGNING_KEY: ${{ secrets.SIGNING_KEY }}
-          VERSION: ${{ needs.outputs.outputs.crux-mir-version }}
+        run: |
+          .github/ci.sh sign ${NAME}.tar.gz
```

Comment:
And here, defensively add double-quotes to specify `"${NAME}.tar.gz"` just in case NAME's value contains spaces or other things that would create issues.

Same comment for the other yaml file.